Tests/XCUITests/FindInPageTest.swift

```diff
@@ -162,13 +162,12 @@ class FindInPageTests: BaseTestCase {
         waitForExistence(app.menuItems["Copy"], timeout: 5)
         // Find in page is correctly launched, bar with text pre-filled and
         // the buttons to find next and previous
-        if app.menuItems["Find in Page"].exists {
-            app.menuItems["Find in Page"].tap()
-        } else {
-            app.menuItems["show.next.items.menu.button"].tap()
-            waitForExistence(app.menuItems["Find in Page"])
-            app.menuItems["Find in Page"].tap()
+        while !app.collectionViews.menuItems["Find in Page"].exists {
```

Comment:
Instead of hard code when "Find in Page" should appear in the context menu, I decided to click ">" (`app.buttons["forward"]`) until I see "Find in Page". If the menu items are reordered in the future, we don't have to update the test as long as "Find in Page" is somewhere in the menu.